content/articles/10.year-review-2025.md

```
@@ -0,0 +1,152 @@
+---
+id: 10
+title: 2025 Year Review
+cover: /articles/10/header.png
+date: January 14, 2026
+description:
+draft: false
+layout: article
+badges: ['#2025', '#review']
+---
+
+![](/articles/10/header.png)
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)